gatling/gatling.py

```diff
@@ -259,42 +274,50 @@ def main():
     # the first argument if it is gatling or manifold.
     my_name, _ = os.path.splitext(os.path.basename(sys.argv[0]))
     for i, arg in enumerate(args):
-        if not i and arg in ('gatling', 'manifold'):
+        if not i and arg in ("gatling", "manifold"):
             my_name = arg
             continue
-        if not arg.startswith('-'):
+        if not arg.startswith("-"):
             try:
                 int(arg)
             except ValueError:
-                args.insert(i, '--')
+                args.insert(i, "--")
                 break
-    opts = docopt(__doc__.replace('{PROGRAM}', my_name), args)
-    if not opts['gatling'] and not opts['manifold']:
-        opts['manifold'] = my_name.startswith('manifold')
-    if opts['-n'] == '#cpu':
-        opts['-n'] = cpu_count()
+    opts = docopt(__doc__.replace("{PROGRAM}", my_name), args)
+    if not opts["gatling"] and not opts["manifold"]:
+        opts["manifold"] = my_name.startswith("manifold")
+    if opts["-n"] == "#cpu":
+        opts["-n"] = cpu_count()
     else:
-        opts['-n'] = int(opts['-n'])
-    opts['-c'] = int(opts['-c'])
-    opts['-m'] = int(opts['-m'])
+        opts["-n"] = int(opts["-n"])
+    opts["-c"] = int(opts["-c"])
+    opts["-m"] = int(opts["-m"])
 
     try:
-        res = distribute(opts['<args>'], opts['-m'], opts['-n'], opts['-c'], opts['manifold'],
-                         opts['-v'])
+        res = distribute(
+            opts["<args>"],
+            opts["-m"],
+            opts["-n"],
+            opts["-c"],
+            opts["manifold"],
+            opts["-v"],
+        )
         errs = [r for r in res if r]
         if errs:
-            sys.exit(f"*** ERROR: There were {len(errs)} errors out of {len(res)} processes.")
-        if opts['-v']:
+            sys.exit(
+                f"*** ERROR: There were {len(errs)} errors out of {len(res)} processes."
+            )
```

Comment:
Why?

Reply:
Yeah, that's a little strange, but I assume that it's due to the limitations of f strings (can't use continuations IIRC).